### DISCUSSION TITLE
[Feature] Use iOS LivePhoto image timezone offset for movie file

### DISCUSSION THREAD

**Discussion by dpantel** [immich-app/immich#4373] [Feature Request]
### The feature

My understanding is that there is a limitation for MOV files, that they don't have the correct timezone offset for the create-date. (I guess that is why there is a TZ setting in the .env file.)

However, since the MOV file is part of the iOS LivePhoto, it has a correlating image file that should have all the proper exif information.

Is it possible to adjust MOV file create date to match the image information on upload? This way, if the storage template uses dates and times, the two file names will match.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
We only recently started incorporating the exif timezone info into the storage template, so I think this is a new bug.

**Reply by dpantel** [reply]
Ok. I just assumed it was a limitation of the MOV file format. But now that I look at the exif info, it does have all the appropriate date-time info, including timezone offsets, as well as GPS.

Should I repost this as a bug?

**Reply by jrasm91** [reply]
#4375
